README.md

```diff
@@ -26,13 +26,23 @@ async function main() {
 
   const aborter = new AbortController();
 
-  const doneP = await rabbitHelper.handleQueue({
-    queueName: "work-queue",
-    handler: async (msg) => {
-      await doWork(msg.body);
-    },
-    signal: aborter.signal,
-  });
+  const doneP = (async () => {
+    // We loop here in case the rabbit channel closes and we need to restart
+    // the queue handler.
+    while (!aborter.signal.aborted) {
```

Comment:
Do we want to include a sleep interval or a retry limit in the sample code here?

Reply:
I think it's fine without